src/Components/CreateImageWizard/formComponents/RepositoriesStatus.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import {
+  CheckCircleIcon,
+  ExclamationCircleIcon,
+  ExclamationTriangleIcon,
+  InProgressIcon,
+} from '@patternfly/react-icons';
+import { Alert, Button, Popover } from '@patternfly/react-core';
+
+type RepositoryStatusProps = {
```

Comment:
I'm still trying to work out best practices for Typescript types, but I think that here it might be good future proofing to use the types provided by the API.

`ApiRepositoryResponse[status]` and `ApiRepositoryResponse[url]`...

Those are both just strings in the current API spec... the url will always be a string but I could imagine the status changing to something like `'valid' | 'failure'`, etc...

Reply:
Completely on board, just still forgetting :no_mouth: